code/modules/admin/admin_verbs.dm

```diff
@@ -801,6 +802,42 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 		can.valve_open = FALSE
 		can.update_icon()
 
+/client/proc/delete_book()
+	set category = "Admin"
+	set name = "Delete Book"
+
+	if(!check_rights(R_ADMIN))
+		return
+
+	var/bookid = input(usr, "What Book ID would you like to remove:", "Literally Fahrenheit 451") as null|num
+	if(!bookid)
+		return
+	
+	var/datum/DBQuery/query_library_print = SSdbcore.NewQuery(
+		"SELECT * FROM [format_table_name("library")] WHERE id=:id AND isnull(deleted)",
+		list("id" = bookid)
+	)
+	if(!query_library_print.Execute())
+		to_chat(usr, "Failed to locate book.")
+		qdel(query_library_print)
+
+	while(query_library_print.NextRow())
+		var/author = query_library_print.item[2]
+		var/title = query_library_print.item[3]
+		var/confirmation = alert(src,"Are you sure you want to delete the book with author [author] and title [title]?","Guy Montag Incarnate","Yes","No")
+		if(confirmation == "Yes")
+			var/datum/DBQuery/query_burn_book = SSdbcore.NewQuery(
+				"UPDATE [format_table_name("library")] SET deleted = 1 WHERE id=:id",
+				list("id" = bookid)
+			)
+			if(!query_library_print.Execute())
+				to_chat(usr, "Failed to delete book.")
+				qdel(query_burn_book)
+			else
+				message_admins("[usr] deleted book number [bookid] with title [title]")
```

Comment:
`log_admin()` it too

Reply:
And give them a chat confirmation that it succeeded

Reply:
> And give them a chat confirmation that it succeeded

They're an admin, they'll see the message_admins